docs/01.CONFIGURATION.md

```diff
@@ -291,6 +291,58 @@ This option is disabled by default.
 | CMake:  | `-DJERRY_MEM_GC_BEFORE_EACH_ALLOC=ON/OFF`    |
 | Python: | `--mem-stress-test=ON/OFF`                   |
 
+### MSVC CRT
+
+This option can be used to enable the use of the MSVC CRT library. This option is disabled by default.
+
+| Options |                                              |
+|---------|----------------------------------------------|
+| C:      | `<none>`                                     |
+| CMake:  | `-DENABLE_STATIC_CRT=ON/OFF`                 |
+| Python: | `--static-crt=ON/OFF`                        |
+
+### Architecture
+
+Specify platform name if supported by generator.
```

Comment:
"I'm not sure about the original intent, but perhaps a minor correction could be applied here as well:
 "Specify the platform name if it is supported by the generator."